app/modals/stacking/stacking-modal.tsx

```diff
@@ -56,47 +76,87 @@ export const StackingModal: FC<StackingModalProps> = ({ onClose }) => {
   // const [loading, setLoading] = useState(false);
   const [blockstackApp, setBlockstackApp] = useState<null | BlockstackApp>(null);
 
-  const { encryptedMnemonic, salt, walletType, publicKey } = useSelector((state: RootState) => ({
-    salt: selectSalt(state),
-    encryptedMnemonic: selectEncryptedMnemonic(state),
-    walletType: selectWalletType(state),
-    publicKey: selectPublicKey(state),
-  }));
+  const { encryptedMnemonic, salt, walletType, publicKey, poxInfo } = useSelector(
+    (state: RootState) => ({
+      salt: selectSalt(state),
+      encryptedMnemonic: selectEncryptedMnemonic(state),
+      walletType: selectWalletType(state),
+      publicKey: selectPublicKey(state),
+      poxInfo: selectPoxInfo(state),
+    })
+  );
 
   const initialStep =
     walletType === 'software'
       ? StackingModalStep.DecryptWalletAndSend
       : StackingModalStep.SignWithLedgerAndSend;
   const [step, setStep] = useState(initialStep);
 
-  const createSoftwareWalletTx = useCallback(async () => {
-    if (!password || !encryptedMnemonic || !salt) {
+  const createSoftwareWalletTx = useCallback(async (): Promise<StacksTransaction> => {
+    if (!password || !encryptedMnemonic || !salt || !poxInfo) {
       throw new Error('One of `password`, `encryptedMnemonic` or `salt` is missing');
     }
     const { privateKey } = await decryptSoftwareWallet({
       ciphertextMnemonic: encryptedMnemonic,
       salt,
       password,
     });
-    console.log({ privateKey });
-    // return makeSTXTokenTransfer({ ...options, senderKey: privateKey });
-  }, [encryptedMnemonic, password, salt]);
+    const txOptions = await poxClient.getLockTxOptions({
+      cycles: numCycles,
+      poxAddress,
+      amountSTX: poxInfo.min_amount_ustx,
```

Comment:
Scope for confusion; isn't clear this prop refers ustx

Reply:
:+1: changed to `amountMicroSTX`